gym_cap/envs/cap_env.py

```diff
@@ -8,6 +8,7 @@
 
 from .agent import *
 from .create_map import CreateMap
+from gym_cap.envs.const import *
```

Comment:
Try not to import const file for now. NUM_BLUE and NUM_UAV should come from the counter.